src/Components/AuthDialog/Views/AuthDialogSignUp.tsx

```diff
@@ -49,15 +49,23 @@ export const AuthDialogSignUp: FC<AuthDialogSignUpProps> = ({
       validateOnBlur={false}
       initialValues={{
         ...INITIAL_VALUES,
-        agreed_to_receive_emails: isAutomaticallySubscribed,
+        agreedToReceiveEmails: isAutomaticallySubscribed,
       }}
       validationSchema={VALIDATION_SCHEMA}
-      onSubmit={async ({ mode, ...values }, { setFieldValue, setStatus }) => {
+      onSubmit={async (
+        { mode, name, email, password, agreedToReceiveEmails },
+        { setFieldValue, setStatus }
+      ) => {
         setStatus({ error: null })
         setFieldValue("mode", "Loading")
 
         try {
-          const { user } = await signUp(values)
+          const { user } = await signUp({
+            email,
+            name,
+            password,
+            agreedToReceiveEmails,
+          })
```

Comment:
Going forward, will avoid passing an object and prefer explicitly destructuring and constructing a payload.